[{"name": "test.test_bicycle_timeseries.BicycleTimeseries.test_calls", "status": "errored", "message": "name 'pd' is not defined", "passed": false, "points": ["p05-08.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 601, in run\n    self.setUp()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/test/test_bicycle_timeseries.py\", line 36, in setUp\n    self.df = bicycle_timeseries()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/src/bicycle_timeseries.py\", line 11, in bicycle_timeseries\n    return pd.DataFrame()\n"]}, {"name": "test.test_bicycle_timeseries.BicycleTimeseries.test_columns", "status": "errored", "message": "name 'pd' is not defined", "passed": false, "points": ["p05-08.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 601, in run\n    self.setUp()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/test/test_bicycle_timeseries.py\", line 36, in setUp\n    self.df = bicycle_timeseries()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/src/bicycle_timeseries.py\", line 11, in bicycle_timeseries\n    return pd.DataFrame()\n"]}, {"name": "test.test_bicycle_timeseries.BicycleTimeseries.test_index", "status": "errored", "message": "name 'pd' is not defined", "passed": false, "points": ["p05-08.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 601, in run\n    self.setUp()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/test/test_bicycle_timeseries.py\", line 36, in setUp\n    self.df = bicycle_timeseries()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/src/bicycle_timeseries.py\", line 11, in bicycle_timeseries\n    return pd.DataFrame()\n"]}, {"name": "test.test_bicycle_timeseries.BicycleTimeseries.test_shape", "status": "errored", "message": "name 'pd' is not defined", "passed": false, "points": ["p05-08.1"], "backtrace": ["  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/anaconda3/lib/python3.6/unittest/case.py\", line 601, in run\n    self.setUp()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/test/test_bicycle_timeseries.py\", line 36, in setUp\n    self.df = bicycle_timeseries()\n", "  File \"/Users/satu/Kurssit/hy-data-analysis-with-python-spring-2019/part05-e08_bicycle_timeseries/src/bicycle_timeseries.py\", line 11, in bicycle_timeseries\n    return pd.DataFrame()\n"]}]